src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -579,6 +594,9 @@ def _configure_security_policy_storage_params(cmd):
         'administrator_login_password': patch_arg_make_required
     })
 
+    c.argument('assign_identity', options_list=('--assign-identity', '-i'),
+                help='Assign an identity for the server.')
```

Comment:
Can you explain this a little more? Imagine I have a general idea of what AAD is, but I don't know how server identity relates to AAD.

Reply:
I can update this to the same help message we use in powershell. What do you think of:
"Generate and assign an Azure Active Directory Identity for this server for use with key management services like Azure KeyVault."